AdornUI-documentation/rating.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" type="image/x-icon" href="./images/adornui-logo.png">
+    <title>AdornUI</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <link rel="stylesheet" href="../AdornUI-Stylesheets/AdornUi.css" />
+</head>
+
+<body>
+    <div class="page-layout">
+        <header class="page-header">
+            <h1 class="lib-name-1">Adorn <span class="lib-name-2">UI</span></h1>
+            <ul class="nav-items-wrapper">
+                <li class="nav-item"><a class="nav-item-link" href="../">Home</a></li>
+                <li class="nav-item"><a class="nav-item-link" href="alert.html">Documentation</a></li>
+            </ul>
+        </header>
+        <aside class="sidebar">
+            <h2 class="sidebar-heading start-guide-text-link">Getting started</h2>
+            <div class="sidebar-items-getting-started">
+                <a class="sidebar-item-link" href="start-guide-installation.html">Installation</a>
+                <a class="sidebar-item-link" href="start-guide-color-palette.html">Colors</a>
+                <a class="sidebar-item-link" href="Text Utilities.html">Typography</a>
+            </div>
+            <h2 class="sidebar-heading">Components</h2><span><i class="fa fa-angle-down"></i></span>
+            <div class="sidebar-items">
+                <a class="sidebar-item-link" href="alert.html">Alerts</a>
+                <a class="sidebar-item-link" href="badges.html">Badges</a>
+                <a class="sidebar-item-link" href="avatar.html">Avatar</a>
+                <a class="sidebar-item-link" href="card.html">Cards</a>
+                <a class="sidebar-item-link" href="button.html">Buttons</a>
+                <a class="sidebar-item-link" href="image.html">Image</a>
+                <a class="sidebar-item-link" href="input.html">Input</a>
+                <a class="sidebar-item-link" href="modal.html">Modal</a>
+                <a class="sidebar-item-link" href="list.html">Lists</a>
+                <a class="sidebar-item-link" href="navigation.html">Navigation</a>
+                <a class="sidebar-item-link" href="navigation.html">Rating</a>
+            </div>
+        </aside>
+        <main>
+            <section class="component-doc-container">
+                <h2 class="component-title">Rating</h2>
+                <h5 class="component-desc">A Star Rating comprises of stars with amount usually at maximum of
+                    five.Ratings are mostly used for ecommerce sites. Users rate an item using stars. </h5>
+                <div class="rating-container">
```

Comment:
I don't see any styling for "rating-container".
You can do away with it if you are not using it in your CSS